### ISSUE TITLE
`@acset` silently ignores nonexistent names

### ISSUE THREAD

**Issue by filonik-cmu** [AlgebraicJulia/ACSets.jl#55]
MWE
```julia
@acset Graph begin V = 2; E = 3; src = [1,1,1]; target = [2,2,2] end
```

**Comment by filonik-cmu** [comment]
Also, @acset silently ignores missing attributes (perhaps intentional), but subsequent migrations fail.

MWE

```julia
g₁ = @acset WeightedGraph{Float64} begin V = 2; E = 3; src = [1,1,1]; tgt = [2,2,2] end
g₂ = @migrate WeightedGraph{Float64} g₁ begin
  V => V
  E => E
  src => src
  tgt => tgt
  Weight => Weight
  weight => weight
end
```

**Comment by jpfairbanks** [comment]
Ohh, it took me a while to understand the problem because I was reading target as tgt. Yeah we should check that all the names exist in the initializer. How urgently do we want to work on this?

**Comment by filonik-cmu** [comment]
For me personally it is not as urgent as some of the other issues, but if it doesn't introduce any unforeseen consequences, I think these kinds of checks would be good for robustness/guarding against user errors.

**Comment by slwu89** [comment]
Closed with #110
